OasisDEX/oasis-borrow
# Basic sell does not take below dust limit

# [Basic sell does not take below dust limit](https://app.shortcut.com/oazo-apps/story/4988/validation-basic-sell-does-not-take-vault-below-dust-limit)

https://www.notion.so/oazo/04329bc9c9ff47deb4ca4b2fefd8923f?v=2b20b6f62e2b4cffa406f3e08049f0ff

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added validation which verify whether basic sell does not take below dust limit
- added min / max slider values handling for basic sell form

  
## How to test 🧪
  <Please explain how to test your changes>
  
- using BasicBS feature toggle flag
- try to set target coll ratio which will result in debt below dust limit and check if error appear (button should be disabled)
- verify according to min / max table from notion whether slider min / max values are correct for different scenarions
